tox.ini

```diff
@@ -37,7 +37,10 @@ commands = python -c "import src.cli.tox_helpers; src.cli.tox_helpers.pyspelling
 [testenv:markdownlint]
 description = Lint markdown files (markdownlint)
 extras = docs
-commands = python -c "import src.cli.tox_helpers; src.cli.tox_helpers.markdownlint()"
+allowlist_externals = bash
+commands = 
```

Comment:
- Ooh, this is a bit wordy in the terminal. Could you add a function to `src/cli/tox_helpers.py` and use this in tox.ini? 
- Also please improve the error message: colon ":" is also accepted as a line ending.
- Additionally, I don't think the regex will catch nested lists due to `^-`.